components/logins/src/login.rs

```diff
@@ -1275,6 +1282,12 @@ mod tests {
                 should_err: false,
                 expected_err: "",
             },
+            TestCase {
+                login: login_with_bad_password,
+                should_err: false,
```

Comment:
"login_with_bad_password" makes the reader think that this case should fail. "login_with_unicode_password" is more descriptive.